src/app/dashboard/ongoingexecutions/ongoingexecutions.component.ts

```diff
@@ -45,6 +46,18 @@ export class OngoingexecutionsComponent implements OnInit {
     return `status-${plugin.pluginStatus.toString().toLowerCase()}`;
   }
 
+  /** canCancelWorkflow
+  /*  calculate if the workflow can be cancelled
+  /*  @param {WorkflowExecution} ongoing - the execution
+  */
+  canCancelWorkflow(ongoing: WorkflowExecution): boolean {
+    return (
+      [`${PluginType.PUBLISH}`, `${PluginType.DEPUBLISH}`].indexOf(
+        ongoing.currentPlugin!.pluginType
+      ) == -1 && !ongoing.cancelling
+    );
+  }
+
```

Comment:
This function is not used (except in test code). I think it needs to be called from the html somewhere?

Reply:
Fixed